src/index.d.ts

```diff
@@ -191,6 +191,11 @@ type SnoozeInformation = {
   snoozeStart: string;
 };
 
+interface DBSnoozeInformation extends SnoozeInformation {
+  userId: string;
+  userName: string;
```

Comment:
Recommended: a separate interface (`UserInformation`?) to encapsulate this data, and be injected into other types as needed (which will be increasingly common).

Potentially `UserActionInformation` (or `AuditInformation`?) then extends `UserInformation` by including a timestamp, which I think is going to be a useful update when we reform the item history.

Reply:
Secondary recommendation: I infer from this that `SnoozeInformation` and `DBSnoozeInformation` appear in different contexts. On the one hand, I would say `DBSnoozeInformation` could have a more informative name, but on the other hand, should we just update the API so that `SnoozeInformation` has the same data wherever it shows up?

Reply:
The `SnoozeInformation` and `DBSnoozeInformation` types are a confusing pattern that I copied over from `Note` and `DBNote`. The distinction is not what is returned from the API but rather the fields needed to create the object vs the fields returned from the API.

going to add a commit to this branch to clean it up a bit